### PR TITLE
Client version

### PR DESCRIPTION
**Pull request by holiman**
Implementation of https://github.com/karalabe/hive/issues/38, this adds client info to all docker images.

### CODE REVIEW

**Comment by holiman** [comment]
Done

File: clients/go-ethereum:local/Dockerfile [hunk 1]
```diff
@@ -17,4 +17,5 @@ RUN chmod +x /geth.sh
 # Export the usual networking ports to allow outside access to the node
 EXPOSE 8545 8546 30303
 
+ADD client.json /client.json
```
Comment (karalabe) [inline clients/go-ethereum:local/Dockerfile]: version.json :P

**Comment by holiman** [comment]
Fixed!

**Pull request merged**